seqr/utils/elasticsearch/es_utils_2_3_tests.py

```diff
@@ -2062,21 +2064,21 @@ def test_multi_project_get_variants_by_id(self):
         )
 
     @mock.patch('seqr.utils.elasticsearch.es_search.MAX_INDEX_NAME_LENGTH', 30)
-    @mock.patch('seqr.utils.elasticsearch.es_search.hashlib.md5')
-    def test_get_es_variants_index_alias(self, mock_hashlib):
+    def test_get_es_variants_index_alias(self):
         search_model = VariantSearch.objects.create(search={})
         results_model = VariantSearchResults.objects.create(variant_search=search_model)
         results_model.families.set(Family.objects.all())
 
-        mock_hashlib.return_value.hexdigest.return_value = INDEX_NAME
         self.mock_es_client.indices.get_mapping.side_effect = lambda index='': {
             k: {'mappings': v} for k, v in INDEX_METADATA.items()}
+        self.mock_search.side_effect = lambda index=None, body=None, **kwargs: create_mock_response(
+            deepcopy(body), index=','.join([INDEX_NAME, SECOND_INDEX_NAME, SV_INDEX_NAME]))
```

Comment:
Please do what I actually asked you to do, this is still much less clear!

Reply:
Okay. Updated.